elife/newrelic-infrastructure-removal.sls

```diff
@@ -0,0 +1,10 @@
+newrelic-infrastructure-configuration:
+    file.absent:
+        - name: /etc/newrelic-infra.yml
+
+newrelic-infrastructure:
+    pkg.purged:
+        - name: newrelic-infra
+
+    service.dead:
```

Comment:
maybe do first `service.dead` and then `pkg.purged`? Although `pkg.purged` may cover everything already